src/java/org/apache/cassandra/io/util/PathUtils.java

```diff
@@ -401,6 +401,18 @@ public static void deleteRecursive(Path path)
         delete(path);
     }
 
+    /**
+     * Empties everything in directory of "path" but keeps the directory itself.
+     *
+     * @param path directory to be emptied
+     * @throws FSWriteError if any part of the tree cannot be deleted
+     */
+    public static void clearDirectory(Path path)
```

Comment:
it feels weird to have production code only for a test. I would move this out of the prod code into test code. Especially concerning because of what this method can do.

Reply:
moved to where it is used.